bundled/umfpack/UMFPACK/Source/umf_grow_front.cc

```diff
@@ -112,7 +112,7 @@ GLOBAL Int UMF_grow_front
 	/* :: frontal matrix size int overflow :: */
 	/* the desired front size is bigger than the integer maximum */
 	/* compute a such that a*a*s < Int_MAX / sizeof (Entry) */
-	double a = 0.9 * sqrt ((Int_MAX / sizeof (Entry)) / s) ;
+	double a = 0.9 * sqrt (((double) Int_MAX / sizeof (Entry)) / s) ;
```

Comment:
Is it intended that you have changed also UMFPACK?

Reply:
Yes, that is an extra commit. The warning is along the lines that the implicit cast to `double` changes the value of `Int_MAX`.